File: EIDSS v6/android.java/workspace/EIDSS/src/com/bv/eidss/model/BaseReference.java
package com.bv.eidss.model;

import android.database.Cursor;
import android.content.ContentValues;

public class BaseReference {
    public long idfsBaseReference;
    public String name;

    @Override
    public String toString()
    {
        return name;
    }

    public static BaseReference FromCursor(Cursor cursor)
    {
    	BaseReference ret = new BaseReference();
        ret.idfsBaseReference = cursor.getLong(cursor.getColumnIndex("idfsBaseReference"));
        ret.name = cursor.getString(cursor.getColumnIndex("name"));
        return ret;
    }

    public ContentValues ContentValues()
    {
    	ContentValues ret = new ContentValues();
        ret.put("idfsBaseReference", idfsBaseReference);
        ret.put("name", name);
        return ret;
    }

}

File: EIDSS v5/android/eidssdroid/Model/BaseReference.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.Database;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

namespace eidssdroid.Model
{
    public class BaseReference
    {
        public Int64 idfsBaseReference { get; set; }
        public String name { get; set; }

        public override string ToString()
        {
            return name;
        }

        #region FromCursor
        public static BaseReference FromCursor(ICursor cursor)
        {
            return new BaseReference()
            {
                idfsBaseReference = cursor.GetLong(cursor.GetColumnIndex("idfsBaseReference")),
                name = cursor.GetString(cursor.GetColumnIndex("name")),
            };
        }
        #endregion

        #region ContentValues
        public ContentValues ContentValues()
        {
            var ret = new ContentValues();
            ret.Put("idfsBaseReference", idfsBaseReference);
            ret.Put("name", name);
            return ret;
        }
        #endregion

    }
}